qmlti/student_nav.php

```diff
@@ -0,0 +1,137 @@
+<?php
```

Comment:
With regards to this comment, I am planning on implementing a proper framework into LTI in the near future. In the meantime, I will separate the business logic + presentation logic with self-defined classes :).